server/base/src/main/java/org/apache/accumulo/server/metadata/ServerAmpleImpl.java

```diff
@@ -343,7 +345,8 @@ public Stream<ScanServerRefTabletFile> getScanServerFileReferences() {
       scanner.setRange(ScanServerFileReferenceSection.getRange());
       int pLen = ScanServerFileReferenceSection.getRowPrefix().length();
       return StreamSupport.stream(scanner.spliterator(), false)
```

Comment:
```suggestion
      return scanner.stream().onClose(scanner::close)
```
This closes the scanner once the stream is closed (can't put the scanner in try-with-resources block here since it would close the scanner before the stream is used).